apps/backoffice-v2/src/common/components/molecules/Map/Map.tsx

```diff
@@ -1,30 +1,28 @@
 import { FunctionComponent } from 'react';
-import L, { LatLngTuple } from 'leaflet';
-import icon from 'leaflet/dist/images/marker-icon.png';
-import iconShadow from 'leaflet/dist/images/marker-shadow.png';
-import { MapContainer, Marker, Popup, TileLayer } from 'react-leaflet';
-import { IMapProps } from './interfaces';
+
+export interface IMapProps {
+  latitude: number;
+  longitude: number;
+  popupContent?: string;
+  className?: string;
+}
 
 export const Map: FunctionComponent<IMapProps> = ({
   latitude,
   longitude,
   popupContent,
   className,
 }) => {
-  const position: LatLngTuple = [latitude, longitude];
-
-  L.Marker.prototype.options.icon = L.icon({
-    iconUrl: icon,
-    shadowUrl: iconShadow,
-  });
-
   return (
-    <MapContainer center={position} zoom={13} className={className}>
-      <TileLayer
-        attribution='&copy; <a href="https://stadiamaps.com/">Stadia Maps</a>, &copy; <a href="https://openmaptiles.org/">OpenMapTiles</a> &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors'
-        url={`https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}{r}.png`}
-      />
-      <Marker position={position}>{popupContent && <Popup>{popupContent}</Popup>}</Marker>
-    </MapContainer>
+    <div
+      className={className}
+      style={{ backgroundColor: '#f0f0f0', padding: '1rem', borderRadius: '0.375rem' }}
+    >
+      <p>Map component (using Leaflet) has been removed to reduce bundle size.</p>
+      <p>
+        Location: {latitude}, {longitude}
+      </p>
+      {popupContent && <p>Details: {popupContent}</p>}
+    </div>
```

Comment:
Is there any plan for when we'd like to re-introduce a map component?

Reply:
we probably will, and revisit the chosen package i guess